cms/datavis/blocks/base.py

```diff
@@ -27,6 +27,9 @@ class BaseVisualisationBlock(blocks.StructBlock):
 
     title = blocks.CharBlock()
     subtitle = blocks.CharBlock()
+    audio_description = blocks.TextBlock(
```

Comment:
I keep going back to this question in my head -- will the field ever be used in other contexts? Would `accessible_description` be more representative?

absolutely non-blocking

Reply:
Sounds good to me, thank you